completions.go

```diff
@@ -270,6 +270,14 @@ func (c *Command) initCompleteCmd(args []string) {
 	}
 }
 
+// SliceValue is a reduced version of [pflag.SliceValue]. It is used to detect
+// flags that accept multiple values and therefore can provide completion
+// multiple times.
+type SliceValue interface {
+	// GetSlice returns the flag value list as an array of strings.
+	GetSlice() []string
+}
```

Comment:
Can you do something like this either here or in the tests?

(This is untested pseudocode written on a phone, please check and adapt)

```suggestion
// SliceValue is a reduced version of [pflag.SliceValue]. It is used to detect
// flags that accept multiple values and therefore can provide completion
// multiple times.
type SliceValue interface {
	// GetSlice returns the flag value list as an array of strings.
	GetSlice() []string
}

var _ SliceValue = (*pflag.SliceValue)(nil)
```

The idea is to validate pflag.SliceValue implements the interface you define

Reply:
@thaJeztah I'm unsure if you saw my comment last week